pkg/config/role.go

```diff
@@ -17,6 +17,9 @@ const (
 
 	// Role_HIGHWAY is for a Highway Node
 	Role_HIGHWAY
+
```

Comment:
Perhaps we should call this roaming? linking is a bit unclear to me as to the state of the device.

Reply:
I decided to go ahead and get rid of this role, not sure why it was there.